### ISSUE TITLE
There are two entries for 154534513 

### ISSUE THREAD

**Issue by jeanconn** [sot/agasc#27]
In the fits version of AGASC 1.6 there are two entries for agasc id 154534513 in this file

n1500/1179.fit

If we make future versions of the fits files, this defect should be repaired.

**Comment by javierggt** [comment]
Was this fixed?

**Comment by jeanconn** [comment]
No.  I'm pretty sure there are still two entries for that star in the fits files.
